src/modifier.ts

```diff
@@ -167,3 +173,178 @@ function pairwiseMerge(modifiers1: Modifiers, modifiers2: Modifiers) {
 export function mergeModifiers(...modifierss: Modifiers[]): Modifiers {
   return modifierss.reduce((a, b) => pairwiseMerge(a, b), {});
 }
+
+function replaceAll(
+  str: string,
+  searchValue: string,
+  replaceValue: string
+): string {
+  const newStr = str.replace(searchValue, replaceValue);
+  if (newStr === str) return newStr;
+  return replaceAll(newStr, searchValue, replaceValue);
+}
+
+/**
+ * Prints the modtrace to the log.
+ * @param modifiers A string (or string[]) containing the modtrace lookup term(s).
+ * @param baseModifier A string where all the info about modifiers in the string[] array can be grabbed with this one lookup term. (Automatically generated in most cases)
+ * Example: printModtrace("Meat Drop") or printModtrace(["Item Drop", "Booze Drop"])
+ */
+
+export function printModtrace(
+  modifiers: string | string[],
```

Comment:
Would it be reasonable to make this be Modifier types instead of strings?

Reply:
I'd leave it as a string, since we can do `modtrace myst` (which mafia does matching for substrings)